docs/_summary.md

```diff
@@ -54,9 +54,12 @@
   * [Bootmagic](feature_bootmagic.md)
   * [Combos](feature_combo)
```

Comment:
```suggestion
  * [Combos](feature_combo.md)
```

Reply:
I don't know why that suggestion thinks the original line starts with three spaces. :confused:

Reply:
Well, it needs one more space :D

Also, this link does work, just not in github.